### PR TITLE
build: publish fatjar on maven by default

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+// grpc for bblfsh/client-scala needs to be newer then in Spark
+libraryDependencies += fixNettyForGrpc
+libraryDependencies += jgit % Compile
+libraryDependencies += siva % Compile
+libraryDependencies += bblfsh % Compile
+libraryDependencies += commonsIO % Compile
+libraryDependencies += enry % Compile
+
+test in assembly := {}
+assemblyJarName in assembly := s"${name.value}-${version.value}.jar"
```
Comment (mcarmonaa) [inline build.sbt]: What is the difference between having all these things grouped in a `lazy val root` variable and having them in this way?

**Comment by bzz** [comment]
Nice! Which artefact gets published now?

AFAIU, it should now work with `--packages` as well as `--jars` without fetching any external dependencies from the Net

File: build.sbt (hunk 1 above)
Comment (erizocosmico) [reply build.sbt]: Moved them to the root of the file while I was testing this and forgot to move them back. If you have only one project I don't think it does make a difference.

I can move it back if you guys want, but IMHO if feels more readable this way

**Comment by erizocosmico** [comment]
@bzz now `engine-{version}.jar` gets published as it used to before, but it is a fatjar instead. Also, we have the `engine-uber.jar` as we used to for easier access with a constant name, which is just a copy of the main fatjar.

**Pull request merged**